src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/AttendanceService.kt

```diff
@@ -451,6 +451,26 @@ class AttendanceService(
       suspended = details.suspended
     )
 
+  private fun toAbsenceDto2(details: OffenderDetails, attendance: Attendance): AbsenceDto =
+    AbsenceDto(
+      attendanceId = attendance.id,
+      bookingId = attendance.bookingId,
+      offenderNo = details.offenderNo,
+      eventId = attendance.eventId,
+      eventLocationId = attendance.eventLocationId,
+      eventDate = attendance.eventDate,
+      period = attendance.period,
```

Comment:
copied to pull the `period` through from the attendance instead.  Didn't want to alter the original as still in use, but will be removed shortly after testing new endpoint